contracts/instruments/RibbonCoveredCall.sol

```diff
@@ -451,7 +492,7 @@ contract RibbonCoveredCall is DSMath, OptionsVaultStorage {
      */
     function maxWithdrawableShares() public view returns (uint256) {
         uint256 withdrawableBalance = assetBalance();
-        uint256 total = lockedAmount.add(assetBalance());
+        uint256 total = lockedAmount.add(withdrawableBalance);
         return
```

Comment:
making sure you saw

358: SWAP_CONTRACT.swap(order); // REVIEW: Note, offchain check needs to be performed for each option to make sure that the whole balance is finally swapped. Otherwise if some oTokens are left, then after exercise some of the profits will be left stuck.

Reply:
We are ignoring that comment because it doesn't make sense, we are going to be selling off otokens in multiple orders so we can't perform such a check.